tests/csv_upload_tests.py

```diff
@@ -351,29 +269,46 @@ def test_import_into_new_postgres(self):
             if sqlalchemy_utils.database_exists(url):
                 sqlalchemy_utils.drop_database(url)
 
+    def test_create_database_duplicate_name(self):
+        db_name = "examples"
```

Comment:
Does this database always exists or could somebody have an installation without it or with a different name?